### DISCUSSION TITLE
Zasob API summary report

### DISCUSSION THREAD

**Discussion by sergeidebug** [allegro/allegro-api#5749] [Suggestions - rejected]
Witam!
Nie mogę uzyskać statystyki podsumowujące za pomocą interfejsu API, ponieważ nie ma takiego zasobu API. 
Proponuję dodać taki zasob API.
Dzięki czemu można oszacować ilość sprzedaży na dzień, miesiąc, tydzień. Kwota zysku. Koszt reklamy. Liczba wniosków, liczba wniosków odrzuconych. I tak dalej

Na przykład takie informacje można uzyskać na swoim koncie osobistym
https://allegro.pl/moje-allegro/sprzedaz/allegro-analytics/podsumowanie ,
ale w tej chwili nie ma możliwości zautomatyzowania ich otrzymania.

**Comment by is-tomaszgasior** [comment]
Czy planujecie wprowadzić możliwość dostępu z poziomu API Allegro do danych dostępnych w tym panelu:
https://allegro.pl/moje-allegro/sprzedaz/allegro-analytics/sprzedaz-dni ?

**Reply by MartaNowaczyk** [reply]
Postaram się to ustalić z zespołem odpowiedzialnym za ten obszar. Jak będę miała jakieś konkretne informacje wrócę z odpowiedzią.

**Comment by PrzemyslawLukanowski** [comment]
Zaktualizowałem status sugestii jako "odrzucona", ponieważ aktualnie nie mamy planów, aby wdrożyć opisaną funkcjonalność w publicznym API.
